apps/copilot-api/entities/subscribtions.entity.ts

```diff
@@ -0,0 +1,21 @@
+import {Entity, JoinColumn, ManyToOne, PrimaryColumn} from "typeorm";
+import {SubscribersEntity} from "./subscribers.entity";
+import {AddressesEntity} from "./addreesses.entity";
+
+@Entity('subscriptions')
+export class SubscriptionsEntity {
+    @PrimaryColumn({type: 'text', name: 'subscriber_id'})
+    readonly subscriber_id!: string;
```

Comment:
No need for suscription_id?

Reply:
I copy this from existing table ant there are only 2 columns

CREATE TABLE IF NOT EXISTS subscriptions (
  subscriber_id TEXT REFERENCES subscribers(subscriber_id),
  address TEXT REFERENCES addresses(address),
  PRIMARY KEY (subscriber_id, address)
);